app/models/piece.rb

```diff
@@ -1,4 +1,38 @@
+# rubocop:disable all
 class Piece < ActiveRecord::Base
   belongs_to :game
   belongs_to :player
+
+  def obstructed?(x_destination, y_destination)
+    pieces = []
+    x = x_position
+    y = y_position
+
+    if moving_horizontally?(x_destination, y_destination)
```

Comment:
Is there a reason for passing in `x_destination` when only `y_destination` is needed for the method?